spec/models/case/scopes_spec.rb

```diff
@@ -94,11 +94,18 @@
 
   describe 'open scope' do
     it 'returns only closed cases in most recently closed first' do
-      Timecop.freeze 1.minute.ago
-      open_case = create :case
-      Timecop.freeze 2.minutes.ago
-      responded_case = create :responded_case
+      open_case, responded_case = nil
+
+      Timecop.freeze(1.minute.ago) do
+        open_case = create :case
+      end
+
+      Timecop.freeze(2.minutes.ago) do
+        responded_case = create :responded_case
+      end
+
       Timecop.return
```

Comment:
I don't think you need the `Timecop.return` if your using the `Timecop.freeeze do....end` form.